yarn-project/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -301,14 +301,27 @@ contract Test {
         storage.example_constant.initialize(&mut note, false);
     }
 
-    #[aztec(public)]
-    fn assert_coinbase(coinbase: EthAddress) {
-        assert(context.coinbase().eq(coinbase), "Invalid coinbase");
+    #[aztec(private)]
+    fn assert_private_global_vars(chain_id: Field, version: Field) {
```

Comment:
If you have `serialize` functions on the global variables you could even match those directly and just pass that in, but likely total overkill. So keep as is.